test/integration/features/basic.feature

```diff
@@ -1,8 +1,31 @@
 @basic
 Feature: Basic
-  In order to use Minishift
-  As a user
-  I need to be able to bring up a test environment
+  As a user I can perform basic operations of Minishift and OpenShift
+
+  Scenario: User can install default add-ons
+    Given Minishift has state "Does Not Exist"
+    When executing "minishift addons install --defaults" succeeds
+    Then stdout should contain
+     """
+     Default add-ons anyuid, admin-user, xpaas installed
+     """
+
+  Scenario: User can enable the anyuid add-on
+    Given Minishift has state "Does Not Exist"
+    When executing "minishift addons enable anyuid" succeeds
+    Then stdout should contain
+     """
+     Addon 'anyuid' enabled
+     """
+
+  Scenario: User can list enabled plugins
+    Given Minishift has state "Does Not Exist"
+    When executing "minishift addons list" succeeds
+    Then stdout should contain
+     """
+     - anyuid         : enabled    P(0)
+     - admin-user     : disabled   P(0)
+     """
```

Comment:
> Another use oc command same way to wait till pv job is done oc get job persistent-volume-setup -n default -o 'jsonpath={ .status.active }'

Sure, except this command does not wait. It just displays the status. So we would need to execute this in a loop. For that we need to extend the infrastructure.

> only issue which I can see without addon is time taken to have pv created which is right now not predicted. 

IMO there are several pros with this. First of all we add some rudimentary tests around add-ons and secondly we have a full deterministic scenario. Without it, we either need to use retry or we need to extend the framework even further to allow execution in a loop until a certain event occurs (I think eventually we will need that as well). Right now, enabling anyuid seems to me the best option.